### DISCUSSION TITLE
Please someone help me please

### DISCUSSION THREAD

**Discussion by Jjr47** [community/community#36324] [Other Feature Feedback, Questions, & Ideas]
Can someone please help me get this stuff off I don’t know how it got on there I don’t wanna mess my phone up please anyone please

**Comment by radar-blip** [comment]
Kindly provide more details and someone might be able to help. 😃

**Comment by Jjr47** [comment]
There is codes or something i do not know where it came from but I read that could mess up my phone and I don’t have no idea what I am doing on here I can read but bad at comprehending but i don’t know what to do
